L/LibSymspg/Compat.toml

```diff
@@ -1,2 +1,2 @@
-[0]
+["0-0.1"]
 julia = "1"
```

Comment:
Please keep this constraint for this release.

Reply:
Thanks, and it is registered again.